### ISSUE TITLE
coding standard should mention that new test modules go into twisted.x.test.test_y and never twisted.test.test_y

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5191]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#5191|
|Type|enhancement|
|Created|2011-07-08 23:00:26Z|
|Branch|https://github.com/twisted/twisted/tree/test-module-arrangement-5191|

It doesn't yet, according to cyli.

<details><summary>Searchable metadata</summary>

```
trac-id__5191 5191
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__branches_test_module_arrangement_5191 branches/test-module-arrangement-5191
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1310166026000000 1310166026000000
changetime__1310334922000000 1310334922000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [32297]) Branching to 'test-module-arrangement-5191'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Thanks for clarifying this, looks good to merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [32307]) Merge test-module-arrangement-5191

Author: exarkun
Reviewer: thijs
Fixes: [#5191](https://github.com/twisted/twisted/issues/5191)

Explicitly document the preferred location of unit tests in the coding standard.
